opts.go

```diff
@@ -0,0 +1,25 @@
+package core
+
+import "time"
+
+const (
+	// DefaultMaxBatchCalls is the default maximum number of calls a batch will take
+	DefaultMaxBatchCalls = 64
```

Comment:
Is this the maximum number of individual calls? i.e. This means that if 64 requests are sent to the batch, that's all it will take?  I'm just thinking in light of the thousands of campaign messages that could go out at once.

Reply:
And I'm guessing we'd just allow for a larger number of batch calls on init if we knew we needed more?

Reply:
64 per batch. more than that, and it will split the remainder into another batch.

a la windows and threadripper.

Reply:
Max is 1024 now. Also, it's configurable - so a controller can set whatever queue it wants :)